### ISSUE TITLE
About Training Data

### ISSUE THREAD

**Issue by paper-coder** [davidnvq/visdial#33]
I run this code, but I cannot get the results in the paper, almost all metrics drop 2 points. I guess it is because the training date is not the same with you. Could you share your training data using Google Drive, OneDirve or BaiduYun, include Image Features (bounding boxes and attributions) and Glove word vectors ? Many thanks.

**Comment by davidnvq** [comment]
Thanks for your interest. Currently, it is possible to share the validation/test features as well as the GloVe vectors. Since the training  feature size is about 98Gb, I don't have any cloud account which suffices to store such data. 
I will check whether there is any medium to upload the training dataset or not. 
Just to confirm, which training features did you obtain from? I obtained the features from Faster R-CNN pretrained on Gnome dataset. I also provided the [extraction code](https://github.com/davidnvq/visdial/blob/master/others/generate_visdial.py) and the step-by-step [instructions](https://github.com/davidnvq/visdial/README.md) on how to obtain the features.

**Comment by davidnvq** [comment]
@paper-coder Except for the above (paid) platforms for sharing the dataset, do you have any other recommendations for uploading the large data?

**Comment by paper-coder** [comment]
Thanks a lot. I used 6B-300dim Glove word vectors, 10-100 adptive Image Features with bounding boxes but without attributes and classes. You can try Google Drive, OneDirve or BaiduYun. If not possible, I often compress the file to .zip and split it into some small files  (Use the command zipsplit) to upload in order to avoid the limitation of a sinlge big file.

**Comment by davidnvq** [comment]
@paper-coder I don't have OneDrive or BaiduYun account. My Google Drive account has a limitation of 15GB. For image features, how did you extract 10-100 adaptive features? Did you follow my code or just extract somewhere else? Models pretrained on Gnome are much better than those pretrained on COCO. I also used `Detectron` Faster R-CNN model pretrained on COCO dataset but the performance decreased several points.

**Comment by paper-coder** [comment]
BaiduYun ordinary free users have 100g free storage space. It is a nice way to store files. When deciding the number of region for each image using a score_thresh, I used the models pretrained on Gnome.

**Comment by davidnvq** [comment]
@paper-coder Thanks for recommending BaiduYun service. I'm still struggling with registering BaiduYun as I'm not Chinese. If you used models pretrained on Gnome but failed to get similar accuracy as reported, I suspected that the extracted features are different. Although the best performance is obtained with image features of 100 objects (per image), the model trained with adaptive features also got comparable performance rather than being lower than 2 points (I've tried myself to reproduce them all). Batch size and other configurations also affect but it should be not that much. 
Meanwhile, I will upload the GloVe vectors but be careful that the order of word vectors must be the same as the code. :D

**Comment by davidnvq** [comment]
If you have one more BaiduYun account, feel free to PM (on Github) me or my email (quanguet@gmail.com). I will upload the extracted features on that account for you. 
Otherwise, please wait for me for a while, I will ask some Chinese friends whether they can help or not.

**Comment by paper-coder** [comment]
Thanks a lot. I will send you my new account.

**Comment by davidnvq** [comment]
I've already uploaded image features for `trainval`, `val`, and `test`, as well as GloVe embeddings and the two pretrained checkpoints for Visdial. Hope that you can make use of them or reproduce the models close to the reported results. 

P/S: If you find this repo and our paper useful, kindly give us a star :3 Many thanks!

**Comment by paper-coder** [comment]
I got the training features. Thanks a lot for your help. I'll try to reproduce the result mentioned in the paper. Many thanks. : )

**Comment by zd11024** [comment]
Hi, @paper-coder !
There were some mistakes when I reproduced this project. Could you share this training data using Baidu Yun, include image features(bbox, attr and cls)? 
Thank you!

**Comment by davidnvq** [comment]
@zd11024 As you said in your email that you already got the data from Feilong @paper-coder, I'd like to close this issue.

@others: Regarding the preprocessed features, I'm asking the university for an account to provide such datasets. Meanwhile, please email me so that I could help or ask for further support from @paper-coder.

**Comment by nishan-chatterjee** [comment]
Hi, are these files still available for download?
"
At the end, the directory $PROJ_ROOT/datasets/bottom-up-attention / should have the following files:

./trainval_resnet101_faster_rcnn_genome_100.h5
./val2018_resnet101_faster_rcnn_genome_100.h5
./test2018_resnet101_faster_rcnn_genome_100.h5
"

**Comment by quangvnai** [comment]
@nishan-chatterjee please kindly shoot me an email to (quanguet@gmail.com), I will still be able to share those files.
